### PR TITLE
choropleth filtering bug when after navigating to a child geography

### PR DESCRIPTION
**Pull request by emre2038**
## Description
Choropleth fills surrounds when changing filter after navigating to a child chography and changing a filter

## Related Issue
https://trello.com/c/DsxofFcc/774-choropleth-fills-surrounds-when-changing-filter-after-navigating-to-a-child-chography-and-changing-a-filter

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally
- [ ] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [ ]  📖 changelog filled out
- [ ] commit messages are meaningful

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: src/js/profile/subindicator_filter.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ export class SubindicatorFilter extends Observable {
     let groupCallback = (selected) => this.groupSelected(selected, subindicatorDd, title);
 
     this.setOptionSelected(indicatorDd, this.selectedGroup, groupCallback);
+    $(indicatorDd).find('.dropdown__list_item.selected').click();
```
Comment (milafrerichs) [inline src/js/profile/subindicator_filter.js:49]: this is ok for now but need to change with the multiple filters refactoring.

**Review by milafrerichs — APPROVED** [review]

**Comment by milafrerichs** [comment]
probably merge with fix from #380 
currently we are in a code freeze on staging.
